<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameOver</class>
 <widget class="QWidget" name="GameOver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>781</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Game Over</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="retryButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>500</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Retry</string>
   </property>
   <property name="autoRepeatDelay">
    <number>300</number>
   </property>
  </widget>
  <widget class="QPushButton" name="highscoresButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>500</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Check High Scores</string>
   </property>
   <property name="autoRepeatDelay">
    <number>300</number>
   </property>
  </widget>
  <widget class="QPushButton" name="mainmenuButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>500</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Return to Main Menu</string>
   </property>
   <property name="autoRepeatDelay">
    <number>300</number>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>480</y>
     <width>500</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Exit Game</string>
   </property>
   <property name="autoRepeatDelay">
    <number>300</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
